dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/UiPluginServiceImpl.java

```diff
@@ -45,6 +54,19 @@ public class UiPluginServiceImpl extends BaseServiceImpl implements UiPluginServ
     @Autowired
     PluginDefineMapper pluginDefineMapper;
 
+    @Autowired
+    private DsVersionDao dsVersionDao;
+
+    private String dsVersion;
+
+    @PostConstruct
+    private void init() {
+        dsVersion = dsVersionDao.selectVersion().map(DsVersion::getVersion).orElse("unknown");
+        if(StringUtils.isBlank(dsVersion)){
+            throw new ServiceException(Status.VERSION_INFO_STATE_ERROR);
+        }
```

Comment:
This is strange, if the t_ds_version is empty will use unknown, but if the version empty will throw exception.